docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.7"
+
+services:
+  peripheral-manager-gpu:
+    build: .
+    restart: on-failure
+    volumes:
+      - /etc/:/etc/
+    environment:
```

Comment:
why do you need these env vars?

Reply:
I don't know. I copied the one in modbus, and adapted. is now removed

Reply:
We need those env variables to access Nuvla through the API

Reply:
not anymore. Check the agent API to know how to register the peripherals